Full KernelSU gki features on non-gki hurray!
Add umount code for KernelSU
fs/namespace.c path_umount [5.10.9]
sake of demo-ing off susfs-implementation

--- a/fs/internal.h
+++ b/fs/internal.h
@@ -89,6 +89,8 @@ extern void __init mnt_init(void);
 extern int __mnt_want_write_file(struct file *);
 extern void __mnt_drop_write_file(struct file *);
 
+int path_umount(struct path *path, int flags);
+
 /*
  * fs_struct.c
